new-site/src/docs/elements/components/text-area/code.mdx

```diff
@@ -0,0 +1,126 @@
+---
+slug: '/elements/components/text-area/code'
+title: 'TextArea - Code'
+---
+
+import { TextArea, Link, IconCheckCircleFill } from 'hds-react';
+import Playground from '../../../../components/Playground';
+
+## Code
+
+### Code examples 
+
+#### Default
+<Playground>
+
+```jsx
+<div style={{ maxWidth: '320px' }}>
+<TextArea
+  id="textarea"
+  label="Label"
+  placeholder="Placeholder"
+  helperText="Assistive text"
+  required
+/>
+<TextArea
+  label="Label"
+  defaultValue="Text area value"
+  helperText="Assistive text"
+  style={{marginTop: 'var(--spacing-s)'}}
+  disabled
+/>
+<TextArea
+  id="textarea-invalid"
+  label="Label"
+  defaultValue="Text area value"
+  errorText="Error text"
+  helperText="Assistive text"
+  style={{marginTop: 'var(--spacing-s)'}}
+  invalid
+/>
+</div>
+```
+
+```html
+<div style={{ maxWidth: '320px' }}>
+<div class="hds-text-input">
+  <label for="textarea" class="hds-text-input__label">
+    Label
+    <span class="hds-text-input__required">*</span>
+  </label>
+  <div class="hds-text-input__input-wrapper">
+    <textarea id="textarea" class="hds-text-input__input" placeholder="Placeholder" required></textarea>
+  </div>
+  <span class="hds-text-input__helper-text">Assistive text</span>
+</div>
+
+<div class="hds-text-input" style={{marginTop: 'var(--spacing-s)'}}>
+  <label for="textarea" class="hds-text-input__label">
+    Label
+  </label>
+  <div class="hds-text-input__input-wrapper">
+    <textarea id="textarea" class="hds-text-input__input" disabled>
+      Text area value
+    </textarea>
+  </div>
+  <span class="hds-text-input__helper-text">Assistive text</span>
+</div>
+
+<div class="hds-text-input hds-text-input--invalid" style={{marginTop: 'var(--spacing-s)'}}>
+  <label for="textarea" class="hds-text-input__label">
+    Label
+  </label>
+  <div class="hds-text-input__input-wrapper">
+    <textarea id="textarea" class="hds-text-input__input">
+      Text area value
+    </textarea>
+  </div>
+  <span class="hds-text-input__error-text">Error text</span>
+  <span class="hds-text-input__helper-text">Assistive text</span>
+</div>
+</div>
+```
+
+</Playground>
+
+#### With tooltip
+<Playground>
+
+```jsx
+<TextArea
+  id="textarea-tooltip"
+  label="Label"
+  placeholder="Placeholder"
+  tooltipLabel="Tooltip label"
+  tooltipButtonLabel="Tooltip button label"
+  tooltipText="Tooltips contain &quot;nice to have&quot; information. Default Tooltip contents should not be longer than two to three sentences. For longer descriptions, provide a link to a separate page."
```

Comment:
There's some problem with the rendering of &quot; :
<img width="322" alt="image" src="https://user-images.githubusercontent.com/2777633/157661885-f2529b64-f681-47e9-aac9-0a621ae01b5c.png">